enhancements/network/user-defined-network-segmentation.md

```diff
@@ -406,15 +406,16 @@ If there are pods still attached to this network, the network will not be remove
+| Role              | Select the network role in the pod, either `Primary` or `Secondary`. Primary network support topologies `Layer2` and `Layer3` only.                                                                                                                                                                                                                                                                    | No       |
+| MTU               | The maximum transmission unit (MTU).<br/>The default is 1400.                                                                                                                                                                                                                                                                                                                                          | Yes      |
+| Subnets           | The subnet to use for the network across the cluster.<br/>E.g. `10.100.200.0/24`.<br/>IPv6 `2001:DBB::/64` and dual-stack `192.168.100.0/24`,`2001:DBB::/64` subnets are supported.<br/>When omitted, the logical switch implementing the network only provides layer 2 communication, and users must configure IP addresses.<br/>Port security only prevents MAC spoofing if the subnets are omitted. | Yes      |
+| ExcludeSubnets    | List of CIDRs.<br/>IP addresses are removed from the assignable IP address pool and are never passed to the pods.                                                                                                                                                                                                                                                                                      | Yes      |
+| JoinSubnets       | Subnet used inside the OVN network topology.  When omitted, this means no opinion and the platform is left to choose a reasonable default which is subject to change over time.                                                                                                                                                                                                                        | Yes      |
+| IPAMLifecycle     | Control IP addresses management lifecycle. When `Persistent` is specified it enable workloads have persistent IP addresses. For example: Virtual Machines will have the same IP addresses along their lifecycle (stop, start migration, reboots). Supported by Topology `Layer2` & `Localnet`.                                                                                                         | Yes      |
+| AddressManagement | Control how much of the IP/MAC configuration will be managed by OVN-Kubernetes. Must be one of `Full`, `MAC`.                                                                                                                                                                                                                                                                                   | Yes      |
```

Comment:
Could we be more specific about what are we exactly managing?

I think we should keep it specific to address assignment and nothing else.

My concern is that if we keep this open for "side effects", such as "`MAC` means MAC spoof filtering enabled", we would be limiting our options to extend the feature set. e.g. in case of MAC spoof filtering, there are multiple strategies how to do it.

My suggestion is to express here that address management is only about assigning addresses from a pool and nothing else.

And if made specific to address assignment, then the `MAC` option does not really make sense. We always have to assign a MAC address.

Reply:
While I agree w/ that, it doesn't exist in a vacuum: the other options are NONE and FULL. 

It makes sense if you *also* look at the other possible values. Having said that, I do think we should try to come up w/ a value name that is self-explanatory and can exist in a vacuum.